lldb-eval/api.h

```diff
@@ -73,6 +98,43 @@ LLDB_EVAL_API
 lldb::SBValue EvaluateExpression(lldb::SBValue scope, const char* expression,
                                  Options opts, lldb::SBError& error);
 
+LLDB_EVAL_API
+ParsedExpression ParseExpression(lldb::SBFrame, const char* expression,
+                                 lldb::SBError& error);
+
+LLDB_EVAL_API
+ParsedExpression ParseExpression(lldb::SBFrame, const char* expression,
+                                 Options opts, lldb::SBError& error);
+
+LLDB_EVAL_API
+ParsedExpression ParseExpression(lldb::SBTarget target, lldb::SBType scope,
```

Comment:
Can we return a `std::shared_ptr<ParsedExpression>` instead? Then the caller can just pass it around and not worry about the lifetime.

Reply:
Done, makes more sense.